### PR TITLE
In app "end of year sale" promos

### PR DESCRIPTION
**Pull request by skyshab**
Last minute in-app "End of year sale" promotions. 

Screenshots: 
Screen Shot 2021-12-07 at 8 19 52 AM

Screen Shot 2021-12-07 at 8 20 03 AM

Screen Shot 2021-12-07 at 12 41 33 PM

### CODE REVIEW

**Comment by borkweb** [comment]
Can we see screenshots of the two notices in place?

File: src/Tribe/Admin/Notice/Marketing/End_Of_Year_Sale.php [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+	/**
+	 * {@inheritDoc}
+	 */
+	public $end_date = 'December 31';
+
+	/**
+	 * {@inheritDoc}
+	 */
+	public function display_notice() {
+		\Tribe__Assets::instance()->enqueue( [ 'tribe-common-admin' ] );
```
Comment (bordoni) [inline src/Tribe/Admin/Notice/Marketing/End_Of_Year_Sale.php:39]:
```suggestion
		tribe_asset_enqueue( [ 'tribe-common-admin' ] );
```

File: src/Tribe/Admin/Conditional_Content/End_Of_Year_Sale.php [hunk 2]
```diff
@@ -0,0 +1,85 @@
+<?php
+namespace Tribe\Admin\Conditional_Content;
+
+use Tribe__Date_Utils as Dates;
+
+/**
+ * Set up for end of year sale promo.
+ *
+ * @since 4.14.7
```
Comment (bordoni) [inline src/Tribe/Admin/Conditional_Content/End_Of_Year_Sale.php:9]: Lets make sure all sinces are TBD

**Review by bordoni — CHANGES_REQUESTED** [review]

File: src/Tribe/Admin/Notice/Marketing/End_Of_Year_Sale.php (hunk 1 above)
Comment (skyshab) [reply src/Tribe/Admin/Notice/Marketing/End_Of_Year_Sale.php]: done

File: src/Tribe/Admin/Conditional_Content/End_Of_Year_Sale.php (hunk 2 above)
Comment (skyshab) [reply src/Tribe/Admin/Conditional_Content/End_Of_Year_Sale.php]: fixed

**Review by bordoni — APPROVED** [review]

**Review by codingmusician — APPROVED** [review]

**Pull request merged**